github/ci/prow/files/job-config.yaml

```diff
@@ -112,3 +148,268 @@ presubmits:
         resources:
           requests:
             memory: "500Mi"
+
+  kubevirt/kubevirtci:
+  - name: check-provision-os-3.11.0-multus
```

Comment:
Can we use yaml inheritance in order to make the config shorter?

Reply:
We can use one or multiple sub directories. Prow can collect all job configs there and combine them. I can create a PR after this one which rearranges that.